package.json

```diff
@@ -36,14 +39,15 @@
     "babel-preset-stage-2": "^6.11.0",
     "file-loader": "^0.10.1",
     "hogan.js": "^3.0.2",
-    "instantsearch.js": "^4.7.2",
+    "instantsearch.js": "4.41.0",
```

Comment:
the latest version is 4.50.0, which will be released in <1hr, which will support high network latency more. A couple other features also are interesting between 4.41 and 4.50

Reply:
Hi @Haroenv as this is only for internal use, and at this moment we are using the instant search v4.41.0 version in Magento AlgoliaBundle.min.js, that is why we fixed it to 4.41.0.

Reply:
ok, I was just suggesting it would be interesting to update :)

Reply:
@Haroenv We have a backlog task to upgrade to the latest instantsearch. Thank you for pointing that out. For now we need this version in the repo as we are using this in Magento AlgoliaSearch extension right now